packages/playwright-core/src/server/injected/injectedScript.ts

```diff
@@ -1166,7 +1166,7 @@ export class InjectedScript {
           throw this.createStacklessError('Element is not a checkbox');
         if (elementState === 'error:notconnected')
           throw this.createStacklessError('Element is not connected');
-        return { received: elementState, matches: elementState };
+        return { matches: elementState };
```

Comment:
seems like this was unused or at least not covered with a test?

Reply:
This sounds strange. If it's not used in Node.js, it is probably used in other languages.

Reply:
will keep it for now.